src/components/atoms/button-social-media.astro

```diff
@@ -0,0 +1,17 @@
+---
+import Icon from './icon';
+
+const { props } = Astro;
+---
+
+<a
+  href={props.name}
```

Comment:
Dlaczego tutaj wykorzystaliśmy `name` jako `href`. Może to być dość mało intuicyjne dla developera, który będzie korzystał z tego komponentu. Dodatkowo przez to, że destrukturyzujemy `props` dopiero linijkę niżej, to jak ktoś przekaże `href`, to nadpisze tę wartość. Wydaje mi się, że najlepiej usunąć tę linijkę, i po prostu zdestrukrutyzować `props`